data/json/npcs/talk_tags.json

```diff
@@ -537,5 +537,81 @@
             "I'm getting my ass out of here!  You'd better do the same, <name_g>!",
             "Fire in the hole, motherfuckers!"
         ]
+    },{
+        "type": "snippet",
+        "category": "<shitty>",
+        "text": [
+            "shitty",
+            "lame",
+            "crappy",
+            "unpleasant",
+            "<swear> <shitty>",
+            "<very> <shitty>",
+            "awful",
+            "terrible",
+            "horrible",
+            "miserable",
+            "fucked-up"
+        ]
+    },{
+        "type": "snippet",
+        "category": "<dumb>",
+            "text": [ 
+            "stupid", 
+            "idiotic", 
+            "dumb", 
+            "dumb-ass", 
+            "moronic", 
+            "mickey mouse", 
+            "shit-for-brains" 
+        ]
+    },{
+        "type": "snippet",
+        "category": "<zombie>",
+        "text": [ 
+            "zombie", 
+            "Z", 
+            "shambler", 
+            "goo-puker", 
+            "walker", 
+            "walking corpse", 
+            "<swear> <zombie>", 
+            "living dead" 
+        ]
+    },{
+        "type": "snippet",
+        "category": "<zombies>",
+        "text": [ 
+            "zombies", 
+            "Z", 
```

Comment:
This should probably be "Zs", or else it sounds like it is just referring to one zombie.

Reply:
Both work, I think, and that could be added. 

"I ran into a bunch of Z back there"
"I can't believe how many Z's we saw"